src/main/java/org/apache/commons/math4/stat/inference/KolmogorovSmirnovTest.java

```diff
@@ -1133,11 +1133,7 @@ private static boolean hasTies(double[] x, double[] y) {
            throw new InsufficientDataException();
        }
 
-       if (values.length == x.length + y.length) {
-           return false;  // There are no ties.
-       }
-
-       return true;
+       return values.length != x.length + y.length;  // There are no ties.
```

Comment:
The comment is then wrong, isn't it? Before it checked if there are no ties (`values.length == x.length + y.length`) and the comment was related to the `false` return value. Now it directly checks if there are ties which is basically the name of the method so that the comment could be simply removed, right?